src/libraries/Microsoft.Win32.SystemEvents/src/Microsoft/Win32/SystemEvents.cs

```diff
@@ -476,7 +437,7 @@ private static void EnsureSystemEvents(bool requireHandle)
                     {
                         // If we are creating system events on a thread declared as STA, then
                         // just share the thread.
-                        if (!UserInteractive || Thread.CurrentThread.GetApartmentState() == ApartmentState.STA)
+                        if (!Environment.UserInteractive || Thread.CurrentThread.GetApartmentState() == ApartmentState.STA)
```

Comment:
Microsoft.Win32.SystemEvents is a NuGet package that targets netstandard2.0. Changing this to use `Environment.UserInteractive` is going to break this NuGet package on .NET Core 2.x and 3.x.

Reply:
oh! - of course